src/Artifacts.jl

```diff
@@ -194,9 +194,9 @@ function topmodel_data_path(; context = nothing)
 end
 
 """
-    lehmann_assouline_or2008_evaporation_data(; context=nothing)
+    lehmann2008_evaporation_data(; context=nothing)
 
-Local path to file containing measured evaporation rate as a function of time
+Returns the path to file containing measured evaporation rate as a function of time
 for bare soil.
```

Comment:
Can mention here that it refers to Figure 8 of the paper.

Reply:
Further down in the docstring it says "Data was originally collected by ... and presented
in Figure 8 of that work."